platform/darwin/src/MGLTypes.h

```diff
@@ -53,11 +54,14 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingMode) {
     MGLUserTrackingModeNone              = 0,
     /** The map follows the user location. */
     MGLUserTrackingModeFollow,
-    /** The map follows the user location and rotates when the heading changes. */
+    /** The map follows the user location and rotates when the heading changes. 
+        The default user annotation shows an indicator  with the current heading, 
+        sized according to the reported accuracy.  */
```

Comment:
Nit: there only needs to be a single space between the period and the end-comment stuff.